lib/url-decorator.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { CompositeDecorator } from 'draft-js';
+import urlRegex from 'url-regex';
+import normalizeUrl from 'normalize-url';
+import { omit } from 'lodash';
+import { viewExternalUrl } from './utils/url-utils';
+
+const urlRe = urlRegex();
+const markdownLinkRe = /\(|\]:\s$/m;
```

Comment:
I'm a bit confused here at what we are wanting to match with this pattern. Could we clarify either through reworking the RegExp or by leaving an explanatory comment?